src/components/LabelValueMetadata.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Renders label/value pair metadata
+ * (e.g. what comes from manifesto) in a dl
+ * @prop {object} labelValuePair
+ */
+class LabelValueMetadata extends Component {
+  /**
+   * render
+   * @return {String} - HTML markup for the component
+   */
+  render() {
+    const { labelValuePairs } = this.props;
+
+    if (labelValuePairs.length === 0) {
+      return (<></>);
+    }
+
+    return (
+      <dl>
+        {labelValuePairs.reduce((acc, labelValuePair, i) => acc.concat([
+          <dt key={`label-${JSON.stringify(labelValuePair.label[0].value)}`}>
```

Comment:
Could use ideas on better ways to generate this key.  The label can be repeated, and that will cause issues here (at least react seems to throw an error in the console, but things still continue to work).

Reply:
React docs: "When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort." So in this case `i` could do it.

Reply:
Ah, okay.  Our linter is failing hard when using the index key, so I may just need to disable it for these lines.  @mejackreed is the preferred method an inline disable here?

Reply:
👍 to disabling here; it seems very, very unlikely the data will change inside of here without having to re-rending the whole thing anyway.

Reply:
👍  sure, inline disable if we need to